zaza/openstack/charm_tests/saml_mellon/tests.py

```diff
@@ -20,28 +20,28 @@
 
 import zaza.model
 from zaza.openstack.charm_tests.keystone import BaseKeystoneTest
-import zaza.charm_lifecycle.utils as lifecycle_utils
 import zaza.openstack.utilities.openstack as openstack_utils
 
 
 class FailedToReachIDP(Exception):
     """Custom Exception for failing to reach the IDP."""
 
-    pass
 
-
-class CharmKeystoneSAMLMellonTest(BaseKeystoneTest):
+class BaseCharmKeystoneSAMLMellonTest(BaseKeystoneTest):
```

Comment:
Changing the name from `CharmKeystoneSAMLMellonTest` to `CharmKeystoneSAMLMellonIDP1Test` will, I think, break stable versions of the charm tests.  We tend to have to leave the old names alone and add new ones only so as to not break existing stable charms.

Reply:
@ajkavanagh

Makes sense. I'll update the PR, and make sure to keep the backwards compatibility.